crates/pop-parachains/src/up/mod.rs

```diff
@@ -206,6 +214,16 @@ impl Zombienet {
 				continue;
 			}
 
+			// Check if command references a parachain template binary without a specified path
+			// (e.g. Polkadot SDK parachain template)
+			if command == "parachain-template-node" || command == "substrate-contracts-node" {
```

Comment:
Fine as is, but could also be `["parachain-template-node", "substrate-contracts-node"].contains(command)` in the future.

Reply:
https://github.com/r0gue-io/pop-cli/pull/297/commits/21d40f281ce9a6088dd08e302bbc73801056f5b3